apps/root/src/pages/strategy-guardian-detail/investment-data/components/expected-returns/index.tsx

```diff
@@ -32,11 +37,23 @@ const ExpectedReturns = ({ userPositions, hidePeriods, isLoading }: ExpectedRetu
           )}
           <ContainerBox flexDirection="column" gap={1} key={period.period}>
             <Typography variant="bodySmallRegular">{intl.formatMessage(period.title)}</Typography>
-            <NetWorthNumber
-              value={earnings[period.period]}
+            <TokenAmount
+              token={isFiat ? emptyTokenWithDecimals(2) : mainAsset}
+              amount={
+                isFiat
+                  ? {
+                      amount: parseUnits(earnings[period.period].total.toFixed(2), 2),
```

Comment:
Is this toFixed(2) needed? For small amounts, like a WBTC srategy, wouldn't it be a problem?

Reply:
this will always be fiat currency :)